README.md

```diff
@@ -1,56 +1,31 @@
-# licence-finder
+# Licence Finder
 
-Licence finder provides the pages that follow https://www.gov.uk/licence-finder (which is served by `frontend`), specifically starting at https://www.gov.uk/licence-finder/sectors.
+Licence Finder is a public facing app that provides the pages that follow https://www.gov.uk/licence-finder (which is served by [frontend][]), specifically starting at https://www.gov.uk/licence-finder/sectors.
```

Comment:
I was surprised to see that this resource is both published and rendered by Licence Finder - I thought only Whitehall did both! Perhaps this introduction can be more informative?

```suggestion
Licence Finder is a service that takes business and activity types and suggests licences you may need to apply for. Its content is published to Publishing API via rake tasks, as it has no admin interface; data is held in the repo as CSVs. Licence Finder renders the frontend of its pages, starting at https://www.gov.uk/licence-finder/sectors. Note that the [start page](https://www.gov.uk/licence-finder) is served by [Frontend][].
```

Reply:
Here's another [frontend publishing app](https://github.com/alphagov/frontend/blob/master/lib/special_route_publisher.rb#L21-L84) @ChrisBAshton  😬

Reply:
Nope, not just Whitehall by a long shot I'm afraid. Nearly all the frontend apps do it - I think there's maybe 2 that don't: https://trello.com/c/hJVUjqbs/197-frontend-apps-publish-content-items-to-publishing-api 

Thanks for the improved description @ChrisBAshton - I've used some of it. I didn't include the Publishing API bit as it didn't seem to capture the situation right, and given it's common, seems unusual to note. I didn't include the CSV data as that seemed misleading given that most of the data is used to populate MongoDB with perhaps a small part used at runtime.